spec.md

```diff
@@ -583,6 +584,22 @@ message VolumeInfo {
 }
 ```
 
+##### CreateVolume Errors
+
+If the plugin is unable to complete the CreateVolume call successfully, it MUST return a non-ok gRPC code in the gRPC status.
+If the conditions defined below are encountered, the plugin MUST return the specified gRPC error code.
+The CO MUST implement the specified error recovery behavior when it encounters the gRPC error code.
+
+Condition | gRPC Code | Description | Recovery Behavior
+| --- | --- | --- | --- |
+| Volume already exists | 0 OK | Indicates that a volume corresponding to the specified volume `name` already exists. | Caller MUST assume the `CreateVolume` call succeeded. |
+| Invalid `name` | 3 INVALID_ARGUMENT | Indicates that the specified volume name is not allowed by the Plugin. More human-readable information MAY be provided in the gRPC `status.message` field. | Caller MUST fix the name before retrying. |
+| Unsupported `parameters` key or value | 3 INVALID_ARGUMENT | Indicates that either one (or more) of the keys in the opaque key/value `parameters` field is not supported by the Plugin and/or one of the values contains invalid data. More human-readable information MAY be provided in the gRPC `status.message` field. This MAY occur, for example, due to caller error, Plugin version skew, etc. | Caller MUST remove the unsupported key/value pair(s) from the list of parameters before retrying. |
```

Comment:
Ditto. Once we have the general "Invalid field in the request", we can probably just remove this row here.

Reply:
Let me know if you agree with my statement above. If you feel strongly, I'm willing to change it.

Reply:
One second thought, I'll go ahead and change it as you described. We can always re-add these in the future, if needed.